nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -366,10 +371,40 @@ define([
                         );
                     }
                 }
-                const defaultSpecModel = specs[appId].makeDefaultedModel();
-                otherParamIds[fileType].forEach((paramId) => {
-                    initialParams[fileType].params[paramId] = defaultSpecModel[paramId];
+
```

Comment:
After setting up the bulk import cell with just the list of non-xSV-parameter files, we push all of those parameters onto the end of the filePathRows.

`inputFileIds` is also used here to get only those parameters that are file inputs (not including the output names). The values of those parameters are added to the list of files that the cell knows about, and is used for verification and validation later on (and by the configure tab on page reloads and so forth).